distribution/docs/src/main/resources/_contents/_endpoints/resource-download-endpoint-contents.adoc

```diff
@@ -0,0 +1,45 @@
+==== Resource Download Endpoint
```

Comment:
@lessarderic, @figliold, @oconnormi is the Resource Download Endpoint an endpoint that should be documented?